CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
-## TBD
-* Updated `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
```

Comment:
@massdosage, @patduin has made a good point that the avro-serde-options transform, when used in a table replication, has this format:

table-replications:
  - source-table:
      blah
    replica-table:
      blah
    transform-options:
      avro-serde-options:
        blah
but when a default is provided, it takes this format:

table-replications:
  - source-table:
      blah
    replica-table:
      blah
avro-serde-options:
  blah
i.e. the "transform-options" qualifier is no longer needed. This PR is based on the assumption that the "transform-options" qualifier should be present.

So we can either merge this and update the avro options to follow the same pattern (and make a breaking change), or we can revert some of the changes here so that we stay in line with how the avro options work.

I'll update the readme in either case and keep the integration test so that it's clear what's going on. Thanks!

Reply:
Putting your comment here so we can more easily reply on it

Reply:
I'm in favour of keeping it the same as the avro-serde-config. I'm not sure adding transform-options adds that much. I think this PR as it is now is already a breaking change right as you've changed some of the public API classes.

Reply:
+1 for keeping it same as `avro-serde-config` coz as a user I would expect if I have provided a global setting, it would take that as default unless another config is specified for that particular `replica-table`.